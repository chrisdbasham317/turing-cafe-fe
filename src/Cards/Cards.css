.section--cards {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  height: 73vh;
  justify-content: space-around;
  margin-top: 25px;
  overflow: scroll;
  width: 100vw;
}