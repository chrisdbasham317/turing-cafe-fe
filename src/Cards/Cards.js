import React from 'react';
import Reservation from '../Reservation/Reservation';
import './Cards.css'



const Cards = (props) => {
  const reservation = props.resData.map(res => {
    return (
      <Reservation
        key={res.id}
        id={res.id}
        name={res.name}
        date={res.date}
        time={res.time}
        number={res.number}
        cxReservation={props.cxReservation}
      /> 
    )
  })
  return (
    <section className='section section--cards'>
      {reservation}
    </section>
  )
}



export default Cards;