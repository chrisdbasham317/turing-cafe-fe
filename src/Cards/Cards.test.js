import React from 'react';
import { shallow } from 'enzyme';
import Cards from './Cards';

describe('Cards', () => {
  let wrapper
  let reservations = [{
    id: 5,
    name: 'John',
    date: '06/17',
    time: '6:30',
    number: 4
  },
    {
    id: 7,
    name: 'Chris',
    date: '06/18',
    time: '6:30',
    number: 5    
  }]
  beforeEach(() => {
    wrapper = shallow(<Cards
      resData={reservations}
    />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
