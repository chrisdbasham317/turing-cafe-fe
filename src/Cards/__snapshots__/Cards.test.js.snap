// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards should match the snapshot 1`] = `
<section
  className="section section--cards"
>
  <Reservation
    date="06/17"
    id={5}
    key="5"
    name="John"
    number={4}
    time="6:30"
  />
  <Reservation
    date="06/18"
    id={7}
    key="7"
    name="Chris"
    number={5}
    time="6:30"
  />
</section>
`;
